configuration.go

```diff
@@ -70,12 +70,11 @@ func (c *ConfigurationService) CheckRequiredServerConfiguration(req *model.Confi
 
 	cfg := c.api.GetConfig()
 
-	mc, err := utils.Merge(cfg, req, nil)
+	mergedCfg, err := config.Merge(cfg, req, nil)
```

Comment:
Maybe it's worth moving https://github.com/mattermost/mattermost/blob/58a7679daa4f463b060e18728216e6b6889caeea/server/config/utils.go#L166 into `public/model` and making it a method of `model.Config`. That way the import of `github.com/mattermost/mattermost/server/v8/config"` becomes unnecessary.